snapshot.go

```diff
@@ -247,17 +247,13 @@ func (s *snapShotManager) downloadSnapshot(scopes []string, snapshot *common.Sna
 	uri := snapshotUri.String()
 	log.Infof("Snapshot Download: %s", uri)
 
-	client := &http.Client{
-		CheckRedirect: func(req *http.Request, _ []*http.Request) error {
-			req.Header.Set("Authorization", "Bearer "+tokenManager.getBearerToken())
-			return nil
-		},
-		Timeout:       httpTimeout,
+	httpclient.CheckRedirect = func(req *http.Request, _ []*http.Request) error {
```

Comment:
As mentioned in the above comment... I think this can be established once during create and leave it alone.